README.md

```diff
@@ -63,3 +77,14 @@ Latest value available for the requested `id`.
 
 To accommodate for decimal values (which are common in prices and rates) this is expected to be multiplied by a power of ten such as `10**9` or `10**18`.
 The specific power of ten MUST explicitly be stated by the `id` used as input.
+
+##### `int status`
+
+Describes the value's status. For consistency the statsus codes will corespond to the codes used for [HTTP](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) status codes 
```

Comment:
```suggestion
Describes the value's status. For consistency the status codes will correspond to the codes used for [HTTP](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes) status codes.
```

Reply:
I personally believe that reusing HTTP codes here will cause more harm than good...
We do not control HTTP status code, and there might be some confusion. If not now, then in the future.

In particular, in a client/server context, 4xx are client errors, and 5xx are server errors. If my smart contract (client) calls an EIP2362 compliant oracle, is the oracle the server? is the API behind the oracle the server? what about oracles hierarchies?
When a result is not (yet) available, should it be 404? 405? 500? 502? 503? Depending on the situation (unknown ID, unavailable parent oracle, ...), they might all make some sens...

Also, If we really want to go with HTTP status code, why use an `int`? HTTP status code are uint between 100 and 599, so a `uint16` is more than enough.

Reply:
Hi Hadrien @Amxx ! I think we are all open to creating our own ADO codes or using something else, this is just one of the ideas that I could recall from one of the calls. Would you like to propose an alternative?